src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days > 1 && days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
+    return days * 40 - 20;
+  } else if (days >= 7) {
+    return days * 40 - 50;
+  }
```

Comment:
Try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content

```suggestion
  if (days > 1 && days < 3) {
    return days * 40;
  } 
  
  if (days >= 3 && days < 7) {
    return days * 40 - 20;
  }
  
  return days * 40 - 50;
```